warp/tests/interop/test_jax.py

```
@@ -1084,6 +1222,76 @@ def test_ffi_jax_callable_graph_cache(test, device):
+
+
+def _run_ffi_jax_cpu_subprocess(test):
+    script = os.path.join(os.path.dirname(__file__), "aux_test_jax_cpu_ffi.py")
+    env = os.environ.copy()
+    env["CUDA_VISIBLE_DEVICES"] = ""
+    env["JAX_PLATFORMS"] = "cpu"
+    env["XLA_FLAGS"] = "--xla_force_host_platform_device_count=2"
+
+    with tempfile.TemporaryDirectory(prefix="warp-jax-cpu-ffi-") as cache_dir:
+        env["WARP_CACHE_PATH"] = cache_dir
+        result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)